x/slashing/signing_info.go

```diff
@@ -0,0 +1,66 @@
+package slashing
+
+import (
+	"encoding/binary"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+// Stored by *validator* address (not owner address)
+func (k Keeper) getValidatorSigningInfo(ctx sdk.Context, address sdk.Address) (info validatorSigningInfo, found bool) {
```

Comment:
def need to update the spec for how this is stored - get's confusing we're now using the validator signing address